### DISCUSSION TITLE
Reference documentation structure could be improved

### DISCUSSION THREAD

**Discussion by crschnick** [jreleaser/jreleaser#1184] [General]
When I read the reference section (https://jreleaser.org/guide/latest/reference/index.html) in the documentation, I usually expect to find everything in there. But some pages, like the one about artifacts (https://jreleaser.org/guide/latest/concepts/artifacts.html) is not listed under reference and instead under concepts, which forces me to go back and forth between sections. It would be nice to have everything relevant additionally listed under reference as well.

**Comment by aalmiray** [comment]
Oddly enough that used to be the case in the beginning. However, given cross-cutting concerns and the importance of certain concepts we decided to move them into their own section as you see them today. 

Perhaps there may be an option to include/cross-link some of the content to minimize browsing back and forth.
